neptune/experiments.py

```diff
@@ -129,36 +130,43 @@ def system_properties(self):
 
             Get experiment system properties.
 
-            >>> experiment.system_properties
+            >>> experiment.get_system_properties
 
         Note:
             The list of supported system properties may change over time.
 
         """
-        return self._simple_dict_to_dataframe(self._leaderboard_entry.system_properties)
-
-    @property
-    def tags(self):
-        return self._leaderboard_entry.tags
+        experiment = self._client.get_experiment(self._internal_id)
+        return self._simple_dict_to_dataframe({
+            'id': experiment.shortId,
+            'name': experiment.name,
+            'created': experiment.timeOfCreation,
+            'finished': experiment.timeOfCompletion,
+            'running_time': experiment.runningTime,
+            'owner': experiment.owner,
+            'size': experiment.storageSize,
+            'tags': experiment.tags,
```

Comment:
Why are tags here but properties are not? What's `system_property` and what is not?

Reply:
List of fields included in system properties was previously defined in: https://github.com/neptune-ml/neptune-client/blob/master/neptune/model.py#L73 and this change use the same set of fields.